debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -74,10 +75,13 @@
     private static final String GET_ALL_CHANGES_FOR_TABLE = "SELECT *# FROM [#db].cdc.[fn_cdc_get_all_changes_#](?, ?, N'all update old') order by [__$start_lsn] ASC, [__$seqval] ASC, [__$operation] ASC";
     protected static final String LSN_TIMESTAMP_SELECT_STATEMENT = "[#db].sys.fn_cdc_map_lsn_to_time([__$start_lsn])";
     protected static final String AT_TIME_ZONE_UTC = "AT TIME ZONE 'UTC'";
-    private static final String GET_LIST_OF_CDC_ENABLED_TABLES = "EXEC [#db].sys.sp_cdc_help_change_data_capture";
+    private static final String GET_LIST_OF_CDC_ENABLED_TABLES = "SELECT s.name AS source_schema, o.name AS source_table, ct.capture_instance, ct.object_id, ct.start_lsn, ct.end_lsn "
+            + "FROM [#db].cdc.change_tables ct "
+            + "LEFT JOIN [#db].sys.objects o ON ct.source_object_id = o.object_id "
+            + "LEFT JOIN [#db].sys.schemas s ON s.schema_id = o.schema_id";
```

Comment:
Sorry, forgot to post it during the review. I believe these should be `INNER JOIN`s. The columns from the joined tables are used to construct the `TableId`: https://github.com/sugarcrm/debezium/blob/520259c0eb68ca85e09de16fb0e216b636e8489f/debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java#L363

It doesn't make much sense to create an identifier from one or two `NULL`s if that's even possible. We can address this while contributing this change upstream.